web/README.md

```diff
@@ -1,9 +1,21 @@
 # fastlane.ci Web app
 
-This project was generated with [Angular CLI](https://github.com/angular/angular-cli) version 1.7.3.
+This project was generated with [Angular CLI](https://github.com/angular/angular-cli) version `1.7.3`.
 
 ## [Development Guidelines and Tutorials](../docs/front_end/README.md)
 
+## Dependencies
+
+### npm
+```
+brew install node
+```
+
+### Frontend dependencies
+```
+npm install
+```
+
 ## Development server
```

Comment:
Do you mind also adding a build prod with the command `npm run build`?

Reply:
Done 👍